src/fundus/publishers/na/the_namibian.py

```diff
@@ -44,5 +48,10 @@ def authors(self) -> List[str]:
 
     class V1_1(V1):
         VALID_UNTIL = datetime.today().date()
-        _paragraph_selector = XPath("//div[contains(@class, 'entry-content')]/p[position()>1]")
-        _summary_selector = XPath("//div[contains(@class, 'entry-content')]/p[position()=1]")
+        _paragraph_selector = XPath("//div[contains(@class, 'entry-content')]/p[(text() or strong) and position()>1]")
+        _summary_selector = XPath("//div[contains(@class, 'entry-content')]/p[(text() or strong) and position()=1]")
+
+        def _base_setup(self, html: str) -> None:
+            html = re.sub(r"(<br>)+", "<p>", html)
+            doc = lxml.html.document_fromstring(html)
+            self.precomputed = Precomputed(html, doc, get_meta_content(doc), get_ld_content(doc))
```

Comment:
Two things here:
1. The intended way to do something like this would be to use the `function` decorator instead of overwriting the `_base_setup`
2. `html` should still reflect the original HTML. We have to find some way around this.

Reply:
I changed it to only modify the html for the purpose of extracting the body. This way it's also not necessary to overwrite the HTML.